lib/report.js

```diff
@@ -158,7 +160,149 @@ class ConciseReporter extends Reporter {
+        index++;
+        if (res.success) {
+          this.message = pad(offset + 2);
+          if (res.type === 'subtest') {
+            const t = res.test;
+            console.log(`${this.message}# Subtest: ${t.caption}`);
+            this.parseTestResults(t, offset + 2);
+            const prefix = t.success ? 'ok' : 'not ok';
+            this.message = pad(offset + 2);
+            let message = `${this.message}${prefix} ${index}`;
+            message += ` - ${t.caption}`;
+            console.log(message);
+          } else {
+            this.checkSuccess(res, index);
+          }
+        }
+        this.lastIndex = index;
+      });
+      if (test.results.length === 0) {
+        this.lastIndex = 0;
+        this.message = pad(offset + 2);
+      }
+      console.log(`${this.message}1..${this.lastIndex}`);
+    }
+  }
+
+  checkSuccess(test, id) {
+    const testPrefix = test.success ? 'ok' : 'not ok';
+    const successfulTest = `${this.message}${testPrefix} ${id}`;
+    console.log(successfulTest);
+  }
+
+  listFailure(test, firstFailure) {
+    test.results.forEach(res => {
+      if (!res.success && firstFailure) {
+        firstFailure = false;
+        this.message += '  ---\n';
+        if (res.message) this.message += `  message: '${res.message}'\n`;
+        this.message += '  severity: ';
+        this.message += res.severity ? `${res.severity}\n` : 'fail\n';
+        this.message += `${pad(2)}data:\n`;
+        this.message += `${pad(4)}got:\n`;
+      }
+      const [expected, actual] = [res.expected, res.actual].map(res =>
+        typeof res === 'function' || res instanceof Error ? res.toString() : res
+      );
+      if (res.test) {
+        this.listFailure(res.test);
+      } else {
+        this.got.push(actual);
+        this.expected.push(expected);
+      }
+    });
+  }
+
+  record(test) {
+    let isSkipped = false;
+    if (!this.successful && !this.failed) console.log('TAP version 13');
+    if (test.success) {
+      this.successful++;
+    } else if (!test.todo) {
+      this.failed++;
+    } else {
+      this.skipped++;
+      isSkipped = true;
+    }
+    const testPrefix = test.success ? 'ok' : 'not ok';
+    let message = `\n${testPrefix} ${this.testNumeration}`;
+    if (test.caption) {
+      const prefix = isSkipped ? ' #SKIP' : test.todo ? ' #TODO' : '';
+      message += ` -${prefix} ${test.caption}`;
+    }
+    console.log(message);
+    this.parseTestResults(test);
+  }
+
+  finish() {
+    const testCounter = this.skipped + this.failed + this.successful;
+    console.log(`1..${testCounter}`);
+    if (this.failed !== 0) {
+      let testHarness = `FAILED test`;
+      testHarness += ` ${this.failedTests.join(', ')}\n`;
+      const percentOfSuccess = (
+        ((testCounter - this.failed) / testCounter) *
+        100
+      ).toFixed(2);
+      testHarness += `Failed ${this.failed}/${testCounter} tests, `;
+      testHarness += `${percentOfSuccess}% okey`;
```

Comment:
what is `'okey'`?

Reply:
For example, 98.63% okey : the standard output in Tap specification when the error is happened the way to display the percent of successful tests

Reply:
My point was, why is it spelled with 'e', not as `okay`?